.github/CODEOWNERS

```diff
@@ -395,6 +395,7 @@
 /core/controllers/question*.py @oppia/lace-backend-reviewers
 /core/controllers/review_tests*.py @oppia/lace-backend-reviewers
 /core/domain/classroom_domain*.py @oppia/lace-backend-reviewers
+/core/domain/classroom_config_domain*.py @oppia/lace-backend-reviewers
```

Comment:
Should we unify the line above, this line, and the line below? `/core/domain/classroom*.py @oppia/lace-backend-reviewers`

Reply:
Done